libs/components/src/lib/components/navigation-menu/components/prizm-navigation-menu/prizm-navigation-menu.component.less

```
@@ -1,6 +1,7 @@
 .prizm-navigation-menu {
   --prizm-navigation-menu__item-padding: 8px 16px;
   --prizm-navigation-menu__item-height: 40px;
+  --prizm-tree-item-height: 40px;
```

Может для поддержки кейса, если кто то переопредлеял высоту айтемов? 
```suggestion
  --prizm-tree-item-height: var(--prizm-navigation-menu__item-height);
```

---

спасибо!